modules/julia/cmake/init.cmake

```diff
@@ -0,0 +1,24 @@
+OCV_OPTION(WITH_JULIA         "Include Julia support (opencv_contrib)"     OFF   IF (NOT ANDROID AND NOT IOS AND NOT WINRT AND NOT WIN32))
+
+ocv_assert(OPENCV_INITIAL_PASS)
+
+if(WITH_JULIA OR DEFINED Julia_FOUND)
+  ocv_cmake_hook_append(STATUS_DUMP_EXTRA "${CMAKE_CURRENT_LIST_DIR}/hooks/STATUS_DUMP_EXTRA.cmake")
+endif()
+
+set(Julia_BUILD "No" PARENT_SCOPE)
```

Comment:
Just having Julia installed is not sufficient for the bindings. The user must also have a source build of the dependencies, which is why I had this extra variable. What should I do instead?

Reply:
Follow OpenCV's naming rules.
Rename: Julia_BUILD => HAVE_JULIA

Also do no turn off variable on this line (to override this from command line if auto-detection doesn't work). Only turn ON this variable below and use it for checks in other places instead of `_FOUND` or similar variables.

Reply:
Done